src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+type Props = {
+  error: string;
+  handleErrorClose: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = props => {
+  const { error, handleErrorClose } = props;
+
+  useEffect(() => {
+    if (error === ErrorTypes.NoErrors) {
+      return;
+    }
+
+    const timeoutId = setTimeout(() => {
+      handleErrorClose();
+    }, 3000);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, [error]);
```

Comment:
React Hook useEffect has a missing dependency: 'handleErrorClose'. Either include it or remove the dependency array